src/runtimes/node/bundlers/esbuild/bundler.ts

```diff
@@ -147,9 +141,40 @@ let require=___nfyCreateRequire(import.meta.url);
       srcFile,
       outputExtension,
     })
+
+    // workaround for https://github.com/evanw/esbuild/issues/3328
+    await Promise.all(
+      Object.keys(metafile.outputs)
+        .filter((filename) => filename.endsWith('.js'))
+        .map(async (filename) => {
+          const content = await readFile(filename, { encoding: 'utf-8' })
+
+          const updated = content.replace(
+            `
+var __glob = (map) => (path) => {
+  var fn = map[path];
+  if (fn)
+    return fn();
+  throw new Error("Module not found in bundle: " + path);
+};
+      `.trim(),
+            `
+var __glob = (map) => (path) => {
+  var fn = map[path] || map[path + '.js'] || map[path + '.json'] || map[path + '/index.js'] || map[path + '/index.json'];
+  if (fn)
+    return fn();
+  throw new Error("Module not found in bundle: " + path);
+};
+            `.trim(),
+          )
+
+          await writeFile(filename, updated, { encoding: 'utf-8' })
+        }),
+    )
+
     const inputs = Object.keys(metafile.inputs).map((path) => resolve(path))
     const cleanTempFiles = getCleanupFunction([...bundlePaths.keys()])
-    const additionalPaths = [...dynamicImportsIncludedPaths, ...includedFilesFromModuleDetection]
+    const additionalPaths = [...includedFilesFromModuleDetection]
```

Comment:
Not sure we need to create a new array now that `includedFilesFromModuleDetection` is the only source?